coronavirus.md

```diff
@@ -0,0 +1,158 @@
+# Responding to the Novel Coronavirus (2019) Pandemic 
+
+Like others in the Great Lakes region and Toronto, we’ve been closely monitoring the Novel Coronavirus 2019 situation. With the pace the situation has evolved since the World Health Organization [declared COVID-19 a pandemic on March 11](https://www.who.int/dg/speeches/detail/who-director-general-s-opening-remarks-at-the-media-briefing-on-covid-19---11-march-2020), we can assume that things will continue to change rapidly in the weeks and months ahead.
+
+In order to stay safe as workers and community members during this we've drafted this document, which contains our pandemic-specific policies and guidance, resources, and information on business and personal supports for responding to COVID-19.
+
+- [Working in the Time of COVID-19](#working-in-the-time-of-covid-19) 
+- [Communication and Education](#communication-and-education) 
+- [Health and Safety Considerations](#health-and-safety-considerations)
+  - [Social Distancing and Self Isolation](#social-distancing-and-self-isolation) 
+  - [Personal Protective Equipment (PPE)](#personal-protective-equipment-ppe) 
```

Comment:
```
- ./_book/coronavirus.html
  *  linking to internal hash #Personal-Protective-Equipment-PPE that does not exist (line 0)

     <a href="#Personal-Protective-Equipment-PPE">Personal Protective Equipment (PPE)</a>
htmlproofer 3.15.2 | Error:  HTML-Proofer found 1 failure!
```

Reply:
Not sure if this one is affected too.

Reply:
this one works I think! been testing locally